src/pretix/control/templates/pretixcontrol/event/fragment_qr_dropdown.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+<ul class="dropdown-menu dropdown-menu-right">
+    <li>
+        <a href="{% url "control:event.qrcode" event=request.event.slug organizer=request.organizer.slug filetype="png" %}{% if url %}?url={{ url|urlencode }}{% endif %}"
```

Comment:
To help IDE highlight code better, please don't use the same quote style when nesting.

Please write

````html
<a href="{% url 'some_view' %}">
````

instead of

```html
<a href="{%url "some_view" %}">
```

Later on, we will look for some lint tool to enforce this.

Reply:
Hi @hongquan, its good idea, but will not work for this case 
![image](https://github.com/fossasia/eventyay-tickets/assets/171931956/ff328c8f-5329-42f6-a646-58b9e87c881d)

Reply:
@odkhang Don't cut the picture, doing so will strip out the context around a pic and people don't understand.
Also, I don't see the code behind the pic, it is possible that the code is incorrect.

Reply:
Also, one advice is to not "embed" too many Django code to the HTML attribute. Code like this is too complex for developer to read and easy to make mistake.

It is better to move the "{% url 'view' %}" out of the `<a href='...'` code, like this:


```django
{% url 'some-url-name' arg arg2 as the_url %}

<a href="{{ the_url }}">I'm linking to {{ the_url }}</a>
```

Reply:
I have moved url view out of a tag as your suggest